app/src/main/res/xml/developer_preferences.xml

```diff
@@ -392,6 +392,10 @@
             android:key="@string/preference_key_show_sequential_user_stats_tooltip"
             android:title="@string/preference_key_show_sequential_user_stats_tooltip" />
 
+        <SwitchPreferenceCompat
+            android:key="@string/preference_key_se_machine_generated_descriptions_tooltip_shown"
+            android:title="seMachineGeneratedDescriptionsTooltipShown" />
+
```

Comment:
Let's also add a developer preference for the ABC group that the user is in, since it will be very useful for testing and QA.